dipy/data/fetcher.py

```diff
@@ -592,16 +595,19 @@ def read_syn_data():
 """
 
 
-def read_mni_template(contrast="T2"):
+def read_mni_template(version="a", contrast="T1"):
```

Comment:
@jchoude It was version "a" which was being fetched when there was only a single version, so that would not cause any issues.